core/src/main/java/org/infinispan/configuration/cache/IndexWriterConfiguration.java

```diff
@@ -14,9 +14,9 @@ public class IndexWriterConfiguration extends ConfigurationElement<IndexWriterCo
    public static final AttributeDefinition<Integer> INDEX_THREAD_POOL_SIZE =
          AttributeDefinition.builder(Attribute.THREAD_POOL_SIZE, 1, Integer.class).immutable().build();
```

Comment:
Isn't this obsolete, since you're using an Infinispan-provided, global thread pool?

Reply:
Hrmm.. now that I more understand how other parts work, I don't know what this option even does, have to confirm myself.

Reply:
Ah, yes it appears we completely ignore this option due to https://github.com/infinispan/infinispan/blob/main/query/src/main/java/org/infinispan/query/concurrent/InfinispanIndexingExecutorProvider.java#L31

@fax4ever maybe we want to wrap in a LimitedExecutor here so the user can actually tune the pool effective size?